src/auxinfo/participant.rs

```diff
@@ -83,6 +85,27 @@ pub enum Status {
 /// # 🔒 Storage requirements
 /// The [`AuxInfoPrivate`] output requires secure persistent storage.
 ///
+/// # High-level protocol description
+/// The auxinfo protocol runs in four rounds:
+/// - In the first round, each participant generates an RSA modulus `N = pq`,
+///   alongside ring-Pedersen parameters `(s, t, λ)` such that `s = t^λ mod N`.
+///   Each participant then produces a zero-knowledge proof `𝚷[prm]` that the
+///   ring-Pedersen parameters are correct. Finally, each participant commits to
+///   the tuple `(N, s, t, 𝚷[prm])` and broadcasts this commitment.
+/// - Once all participants have received all broadcasted commitments, in the
+///   second round each participant sends its own decommitment to all other
+///   participants.
+/// - In the third round, each participant (1) checks the validity of all the
+///   commitments plus the validity of the committed `𝚷[prm]` proof, and (2)
+///   produces two additional proofs: the first, `𝚷[mod]`, which asserts the
```

Comment:
We should fix this documentation, since it's not just a single `𝚷[fac]` proof, but instead one for each of the other participants.

Reply:
Good catch. Fixed, alongside a larger rewording to hopefully make this section clearer.